src/image-viewer.js

```diff
@@ -339,17 +340,25 @@ export class ImageViewer {
       const deltaX = this.viewerBox_.width / 2 - e.data.clientX;
       const deltaY = this.viewerBox_.height / 2 - e.data.clientY;
       this.onZoom_(newScale, deltaX, deltaY, true).then(() => {
-        return this.onZoomRelease_(0, 0, 0, 0, 0, 0);
+        return this.onZoomRelease_();
```

Comment:
Refactored this, since `DoubleTap` zoom and `Pinch` zoom only needs a subset of the `onZoomRelease` functionality necessary for `TapZoom`.